Esrb filtering

pjthomso opened the pull request.

pjthomso committed:
Converted ESRB dict to list of structs

pjthomso committed:
merge from review merge branch

pjthomso committed:
Merge remote-tracking branch 'refs/remotes/origin/master' into ESRB-filtering

pjthomso committed:
Checking user age to view game details

Tested

pjthomso committed:
Began js function for checking age of visitors

pjthomso committed:
script for checking visitor age

-functional, but not persisting the age yet, will fix this

pjthomso committed:
Fine-tuning age checking script and controller logic

-added cookie to persist visitor age (1 year expiration)
-added loading.gif and changed star rating css to fix an error caused by
not having the image at the path in the css

pjthomso committed:
fixed reviews rating page

-Star rating was showing clear button when it was supposed to be read
only

pjthomso committed:
Merge remote-tracking branch 'refs/remotes/origin/master' into ESRB-filtering

BlackthornYugen commented:
+1

ndahlquist-cc commented on SeaSharpe_CVGS/SeaSharpe_CVGS/Controllers/GameController.cs:
```diff
@@ -1,150 +1,218 @@
+        /// Employee side - list all games
+        /// Member side - search feature, list matching games
+        /// </summary>
+        /// <returns>list of games view</returns>
```
the <returns> content needs updating?

ndahlquist-cc commented on SeaSharpe_CVGS/SeaSharpe_CVGS/Controllers/GameController.cs:
```diff
@@ -1,150 +1,218 @@
-            if(nameSearch != null)
-            {
+
+        /// <summary>
```
needs params in the doc comments?

ndahlquist-cc commented on SeaSharpe_CVGS/SeaSharpe_CVGS/Controllers/GameController.cs:
```diff
@@ -158,207 +226,207 @@ public ActionResult GameManagement()
+        /// <returns>view of games' list</returns>
+        //ADD EMPLOYY AUTH
+        [HttpPost]
         [ValidateAntiForgeryToken]
```
params should include platform/category? remove note to add employee auth

ndahlquist-cc commented on SeaSharpe_CVGS/SeaSharpe_CVGS/Controllers/GameController.cs:
```diff
@@ -158,207 +226,207 @@ public ActionResult GameManagement()
+        /// Employee side - edit game
+        /// </summary>
+        /// <param name="id">game id</param>
         /// <returns>add/edit game view</returns>
```
should only be Edit game view

ndahlquist-cc commented on SeaSharpe_CVGS/SeaSharpe_CVGS/Controllers/GameController.cs:
```diff
@@ -158,207 +226,207 @@ public ActionResult GameManagement()
+        /// </summary>
+        /// <param name="game">game object</param>
+        /// <returns>list of games view</returns>
+        [HttpPost]
```
add category param

BlackthornYugen commented:
@pjthomso My +1 still stands and since all of the feedback was on doc comments this should be good to merge right? 

When you have time could you merge master into your branch, make sure it still builds and then merge to master?

pjthomso committed:
updated from master

pjthomso committed:
Updated doc comments

pjthomso commented:
updated from master and fixed the doc comments, It builds and runs but i'm at school and having uses updating the database using package manager so I wasn't able to double check the pages which had conflicts to ensure they still looked proper.

pjthomso committed:
merging from master

pjthomso commented:
Updated from master again -.-
